.github/workflows/docs.yml

```diff
@@ -7,8 +7,8 @@ jobs:
 
     runs-on: ubuntu-latest
     # available list of containers here:
-    # https://hub.docker.com/r/unityci/editor/tags?page=1&ordering=last_updated&name=ubuntu-2020.1.17f1-base
-    container: unityci/editor:ubuntu-2020.1.17f1-base-0.10.0
+    # https://hub.docker.com/r/unityci/editor/tags?page=1&ordering=last_updated&name=ubuntu-2020.3.4f1-base
+    container: unityci/editor:ubuntu-2020.3.4f1-base-0.11.0
```

Comment:
cache key needs to be set to `key: Library-2020.3.4`  for this action

Reply:
Nice find thanks, fixed